apps/src/templates/alert.jsx

```diff
@@ -16,19 +18,28 @@ var Alert = React.createClass({
         position: 'absolute',
         zIndex: 1000,
         top: 45,
-        left: 350,
+        left: this.props.leftMargin,
```

Comment:
Oooh, I kind of like the idea of having a container like that live in displayAlert, sine we already have some logic that creates the necessary DOM container there.